.travis.yml

```diff
@@ -209,6 +209,14 @@ jobs:
       script:
         - travis_retry ./scripts/if_changed.sh ./scripts/build.sh $PROJECT $PLATFORM $METHOD
 
+    # Validate all Cocoapods configurations
+    # This may take long time, so we would like to run it only once all other tests pass
+    - stage: cocoapods_compatibility_check
+      name: Validate all Cocoapods configurations 
+      script:
+        # TODO: ./scripts/if_changed.sh to ./scripts/if_cron.sh once testing finished
```

Comment:
I would like to test it together with other tests before merging. I'll change it once it's tested.